pints/_mcmc/__init__.py

```diff
@@ -878,6 +932,37 @@ def samplers(self):
         """
         return self._samplers
 
+    def _sample_x0(self, active, evaluator):
+        """
+        Samples an initial starting point for the chains using the supplied
+        `init` function.
+        """
+        initialised_finite = False
+        init_fn = self._init_fn
```

Comment:
Would pass this as an argument, along with the other args

Reply:
It's tricky because the initialisation function is defined during instantiation. So, I'd end up passing it to run (or calling it from within run) then passing to _sample_x0. So, either case, I think it'd have to be a self_.init_fn job no?